plonky2/src/gadgets/ecdsa.rs

```diff
@@ -72,7 +74,15 @@ mod tests {
 
         type Curve = Secp256K1;
 
-        let config = CircuitConfig::standard_ecc_config();
+        const WIDE: bool = true;
```

Comment:
Should we have two different tests maybe?

Reply:
Good idea, done